### ISSUE TITLE
Colorize new titles in the articles list -> Add "new" flag to articles

### ISSUE THREAD

**Issue by martinrotter** [martinrotter/rssguard#1461]
Add "is new" flag to newly fetched articles. The flag will be turned off once article is marked as read or feed is fetched once more.

Also add to article highlighter "Highlight new articles" and to filterer "Show new articles only".

### Discussed in https://github.com/martinrotter/rssguard/discussions/1460

<div type='discussions-op-text'>

<sup>Originally posted by **scottyme** July 29, 2024</sup>
I love the blue colorization of the feeds with new articles in the feeds list. Can new titles in the articles list be colorized also?</div>

**Comment by EmJotGeh** [comment]
yes, it worked somehow once, but I have to search for the setting all the time. The setting for the colors would be better under the tab Settings - Feeds & Articles - Articlelist. This way, setting the colors is not intuitive.
